### PR TITLE
Fixes InvalidEntityException provoked by entityOldValue string > 65k

### PR DESCRIPTION
**Pull request by noriam**
60k is more than enough

Here are the related JIRA issues:
https://assoconnect.atlassian.net/browse/AN-14257
https://assoconnect.atlassian.net/browse/AN-14352

Here are the multiple Sentry entries: https://assoconnect.sentry.io/issues/?project=270186&query=is%3Aunresolved+65535&referrer=issue-list&statsPeriod=90d

### CODE REVIEW

File: src/Serializer/LogSerializer.php [hunk 1]
```diff
@@ -73,11 +74,13 @@ private function formatField(object $entity, string $field): mixed
     private function formatValue($value): mixed
     {
         switch (gettype($value)) {
+            case 'string':
+                return self::MAX_STRING_LENGTH < strlen($value)
```
Comment (sylfabre) [inline src/Serializer/LogSerializer.php]: This is not tested

Comment (sylfabre) [inline src/Serializer/LogSerializer.php]: Using a ternary is useless: you can always call `mb_substr()`

And you're mixing regular vs multi bytes functions here 😉

File: src/Serializer/LogSerializer.php [hunk 2]
```diff
@@ -73,24 +73,13 @@ private function formatField(object $entity, string $field): mixed
...
-                return array_map(__METHOD__, $value);
-            default:
-                throw new \DomainException('Unhandled type');
-        }
+        return match (gettype($value)) {
+            'string' => mb_substr($value, 0, self::MAX_STRING_LENGTH),
+            'NULL', 'boolean', 'double', 'integer' => $value,
+            'object' => $this->formatObject($value),
+            'array' => array_map(__METHOD__, $value),
+            default => throw new \DomainException('Unhandled type'),
```
Comment (sylfabre) [inline src/Serializer/LogSerializer.php]: Useless with a match!

Comment (noriam) [reply src/Serializer/LogSerializer.php]: Pas de default est KO pour phpstan: Match expression does not handle remaining values: 'resource'|'resource (closed)'|'unknown type'

**Pull request merged**